homeassistant/components/plugwise/sensor.py

```diff
@@ -94,13 +120,15 @@
         native_unit_of_measurement=POWER_WATT,
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
```

Comment:
Why is it disabled?

Reply:
This is mainly for Plugs. There are not many Plugs that show produced power other than 0.

Reply:
> There are not many Plugs that show produced power other than 0.

What does that mean?

Reply:
There are not many single devices that produce power instead of consume it.

Reply:
Right, but still a primary entity, so that makes sense in a way 👍